index.js

```diff
@@ -99,7 +100,7 @@ function Primus(server, options) {
   }
 
   this.parsers(options.parser);
-  this.initialise(options.transformer || options.transport, options);
+  this.initialise(options.transformer, options);
```

Comment:
Technically this is a breaking change but the option has never been documented so I think we should be fine.